src/scenario/no_extra.rs

```
@@ -0,0 +1,37 @@
+use serde::{
+    de::{self, Visitor},
+    Deserialize, Serialize,
+};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize)]
+pub struct NoExtra;
+
+impl<'de> Deserialize<'de> for NoExtra {
```

This validation has blind spot - duplicate keys in the YAML are silently overridden. For example, in `bind-node.yaml`:
```
events:
  - id: send
    send:
      from: dummy
      type: V
      data:
        exact:
          one: vienas
          two: du
          three: trys
          two: NO_DU_FOR_YOU
```
Produces:
`bind $PAYLOAD <- Object {"one": String("vienas"), "three": String("trys"), "two": String("NO_DU_FOR_YOU")}`
Because the second two silently replaces the first, test authors can waste time chasing these mistakes.
Negative tests (e.g. in `tests/scenario_invalid.rs`) would be great to ensure that duplicate fields cause a clear failure.
See also the related serde-yaml issue with duplicates: https://github.com/dtolnay/serde-yaml/issues/150